### ISSUE TITLE
Exception on Payment Request

### ISSUE THREAD

**Issue by lippoliv** [btcpayserver/btcpayserver#672]
**Describe the problem/bug**

I created an Payment Request over `0.00060004 BTC`.
I can open it in the Browser.
While pressing "Pay", nothing happens.

Tested in Edge and Chrome.

**Your environment**
* Version of BTCPay Server: v1.0.3.81
* Deployment method: Docker (generated)
* Other relevant environment details: 

**Client Logs**
```
 {description: "An unexpected error occurred invoking 'Pay' on the server.", message: "An unexpected error occurred invoking 'Pay' on the server.", stack: "Error: An unexpected error occurred invoking 'Pay' on the server. at t.callbacks[] (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:17432) at n.prototype.processIncomingData (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:18521) at connection.onreceive (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:14825) at Anonymous function (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:44191) at s (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:40951) at Anonymous function (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:40241) at o (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:39:39973) at Anonymous function (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:6:145566) at Anonymous function (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:6:145437) at h (https://pay.lipperts-web.de/bundles/payment-request-bundle-1.min.js?v=9PoXkCmc-B9nIjymn-H0f9nBwoP3Hu1k8_PzGG1XOLU:6:105606)"}
```

**Server Logs**
```
2019-03-11 06:41:21.786 +00:00 [ERR] Failed to invoke hub method 'Pay'.
System.InvalidOperationException: No service for type 'Microsoft.AspNetCore.Mvc.ViewFeatures.ITempDataDictionaryFactory' has been registered.
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
   at Microsoft.AspNetCore.Mvc.Controller.get_TempData()
   at Microsoft.AspNetCore.Mvc.Controller.View(String viewName, Object model)
   at Microsoft.AspNetCore.Mvc.Controller.View(Object model)
   at BTCPayServer.Controllers.PaymentRequestController.ViewPaymentRequest(String id) in /source/Controllers/PaymentRequestController.cs:line 223
   at BTCPayServer.Controllers.PaymentRequestController.PayPaymentRequest(String id, Boolean redirectToInvoice, Nullable`1 amount, CancellationToken cancellationToken) in /source/Controllers/PaymentRequestController.cs:line 232
   at BTCPayServer.PaymentRequest.PaymentRequestHub.Pay(Nullable`1 amount) in /source/PaymentRequest/PaymentRequestHub.cs:line 47
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.ExecuteHubMethod(ObjectMethodExecutor methodExecutor, THub hub, Object[] arguments)
   at Microsoft.AspNetCore.SignalR.Internal.DefaultHubDispatcher`1.Invoke(HubMethodDescriptor descriptor, HubConnectionContext connection, HubMethodInvocationMessage hubMethodInvocationMessage, Boolean isStreamedInvocation)
```

**To Reproduce**
1. Go to 'Payment Requests'
2. Click on 'Create a new payment request'
3. Set the amount (`0.00060004 BTC` or `0.0009001 BTC` for example)
4. Create it
5. Open it (View)
6. Click on "Pay"

**Expected behavior**
I get an payment Dialog **OR** an proper warning.

**Actual behavior**
Nothing happens (except the logs)

**Screenshots/Links**

**Additional context**

_Regards, lippertsweb_

**Comment by NicolasDorier** [comment]
oh, this is strange. Taking a look.

**Comment by NicolasDorier** [comment]
works on my server. Will find out

**Comment by lippoliv** [comment]
Thank you. Small Addition: I was able to pay them using the "Pay" button from the backend.

**Comment by NicolasDorier** [comment]
Can you try to update? I don't think it will fix it though but give a try.

**Comment by NicolasDorier** [comment]
Releasing 1.0.8.82 soon... shoudl fix it

**Comment by NicolasDorier** [comment]
@lippoliv released 1.0.8.83, please give it a try.

**Comment by lippoliv** [comment]
@NicolasDorier Thanks for the fast help. It works now :)
